pyproject.toml

```diff
@@ -1,7 +1,7 @@
 [build-system]
 requires = [
     "setuptools>=36.6",
-    "packaging",
+    "packaging<=21.3",
```

Comment:
We can just replace the code in setup.py as was done in qutip/qutip@2fc35ba8c6 instead of pinning packaging.